docs/ddl-of-carbondata.md

```diff
@@ -1095,18 +1095,27 @@ Users can specify which columns to include and exclude for local dictionary gene
   about current cache used status in memory through the following command:
 
   ```sql
-  SHOW METADATA
+  SHOW METACACHE
   ``` 
   
   This shows the overall memory consumed in the cache by categories - index files, dictionary and 
   datamaps. This also shows the cache usage by all the tables and children tables in the current 
   database.
   
   ```sql
-  SHOW METADATA ON TABLE tableName
+  SHOW METACACHE ON TABLE tableName
```

Comment:
Please update these points in the section
1. Dictionary cache size is 0 when dictionary include column is not included in where clause in select filter query
2. Cache information is updated only after the select query is executed. In case of alter table for altering metacache properties the previous cache is invalidated after the next select query is executed.